### PR TITLE
Ergo native token

### PR DESCRIPTION
**Pull request by catena2w**

### CODE REVIEW

File: papers/whitepaper/currency.tex [hunk 1]
```diff
@@ -1,51 +1,41 @@
...
+like the known 51\% attack.
+
+\Erg{} emission will be finished within just eight years, and after that miner will only receive \Erg{} from
+fees.
+Ergo block size and block computation cost are limited, and thus miners are enforced to
+choose only a subset of transactions from mempool during high load.
+Fees will help miners to sort the transactions, preventing spam attacks while allowing
+to include transactions from honest users in blocks.
+
+Besides network and computation resources, transaction utilizes storage by increasing state size.
```
Comment (kushti) [inline papers/whitepaper/currency.tex]: a transaction

File: papers/whitepaper/currency.tex [hunk 2]
```diff
@@ -1,51 +1,41 @@
...
+In existing cryptocurrencies, an element of the state once created lives possibly forever without
+paying anything for that, leading to continuously increasing state size.
+In contrast, Ergo has a storage rent component, that periodically charges users for every byte
```
Comment (kushti) [inline papers/whitepaper/currency.tex]: no comma is needed before "that"

File: papers/whitepaper/currency.tex [hunk 3]
```diff
@@ -1,44 +1,41 @@
...
-All \Erg{} tokens that will ever circulate in the system are present in
-the initial state and are divided into 3 parts~(boxes):
-
-\knote{no any notion of a box was introduced before, the introduction is sudden}
-\dnote{I propose to add section about Ergo state before this one and introduce box notion in it}
+As most of the blockchain systems, Ergo platform have its native token which is
+called \Erg{} and is divisible to up to $10^9$ smallest units, \nanoErg{}s.
+\Erg{} provides stability to Ergo platform by several meaningful applications.
+
+During the initial phase of Ergo live, miners will receive the reward in \Erg{}s created
```
Comment (kushti) [inline papers/whitepaper/currency.tex]: "live" => life, "an initial phase"

File: papers/whitepaper/currency.tex [hunk 4]
```diff
@@ -1,44 +1,41 @@
...
+out of thin air according to a predefined and hard-coded token emission schedule
```
Comment (kushti) [inline papers/whitepaper/currency.tex]: Not out of thin air, but out of a miners rewards box protected by an emission script.

File: papers/whitepaper/currency.tex [hunk 5]
```diff
@@ -1,44 +1,41 @@
...
+~(see~\ref{sec:emission} for more details).
+This coins will incentivize miners to participate in the Ergo network, securing it from hashrate-based attacks
+like the known 51\% attack.
```
Comment (kushti) [inline papers/whitepaper/currency.tex]: 50+%, and what attack is considered here concretely? Double-spending via chain reorg?

File: papers/whitepaper/currency.tex [hunk 6]
```diff
@@ -1,44 +1,41 @@
...
+Thus, being a platform for contractual money, Ergo is suitable to build applications and monetary systems
+on top of it.
+However, participating in such systems would require to use Erg, making the network more stable and secure.
+
+\subsection{Emission}
+\label{sec:emission}
+
+
+All \Erg{} tokens that will ever circulate in the system are presented in the initial state and
+that will consists of 3 boxes:
```
Comment (kushti) [inline papers/whitepaper/currency.tex]: "initial state, which consists of 3 boxes"

**Comment by catena2w** [comment]
I haven't deleted text answering the question "Why one needs to have Ergs":
```
Thus, being a platform for contractual money, Ergo is suitable to build applications and monetary systems on top of it. However, participating in such systems would require to use Erg, making the network more stable and secure.
```

File: papers/whitepaper/currency.tex (hunk 5 above)
Comment (catena2w) [reply papers/whitepaper/currency.tex]: Added a link to https://en.bitcoinwiki.org/wiki/51%25_attack

**Pull request merged**